Author: Steffen Moeller
Description: Allowing descriptions of hosts, vendor or model that are larger than a SMS.
--- a/sched/sched_driver.cpp
+++ b/sched/sched_driver.cpp
@@ -53,9 +53,9 @@
 using std::vector;
 
 struct HOST_DESC{
-    char os_name[256];
-    char p_vendor[256];
-    char p_model[256];
+    char os_name[512];
+    char p_vendor[512];
+    char p_model[512];
 };
 
 vector<HOST_DESC> host_descs;
